HW3.postman_collection.json

```diff
@@ -0,0 +1,133 @@
+{
+	"info": {
+		"_postman_id": "0860eec4-d24b-49d5-9c82-45a6ebb7146f",
+		"name": "HW_3",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "21168131"
+	},
+	"item": [
+		{
+			"name": "Pokemon_1",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"const jsonObj = pm.response.json();\r",
+							"const nameArray = [];\r",
+							"jsonObj.results.forEach(result => {\r",
+							"    const newObj = {};\r",
+							"    newObj.name = result.name;\r",
+							"    newObj.url = result.url;\r",
+							"    nameArray.push(JSON.stringify(newObj));\r",
+							"\r",
+							"});\r",
+							"pm.globals.set(\"nameArray\", JSON.stringify(nameArray));\r",
```

Comment:
It's better to use collection or env variables instead of global.
Postman note: [link](https://learning.postman.com/docs/sending-requests/variables/#variable-scopes)